pkg/reconciler/revision/resources/meta.go

```diff
@@ -18,17 +18,31 @@ package resources
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
 	"knative.dev/pkg/kmeta"
 	"knative.dev/serving/pkg/apis/serving"
 	v1 "knative.dev/serving/pkg/apis/serving/v1"
 )
 
+var (
+	excludeLabels = sets.NewString(
+		serving.RouteLabelKey,
+		serving.RoutingStateLabelKey,
+		serving.RevisionLabelKey,
```

Comment:
nit: revision key is added back when we make the labels - having it here may be confusing